runtests.jl

```diff
@@ -13,32 +13,37 @@ macro test_broken(ex)
     @test eval(current_module(), ex)
 end
 
-for (root, dirs, files) in walkdir("exercises")
-    for exercise in dirs
-        # Allow only testing specified execises
-        if !isempty(ARGS) && !(exercise in ARGS)
-            continue
-        end
+if !isdir("tmp/")
+    mkdir("tmp/")
+end
+
+for exercise in readdir("exercises")
+    # Allow only testing specified execises
+    if !isempty(ARGS) && !(exercise in ARGS)
+        continue
+    end
+
+    exercise_path = joinpath("exercises", exercise)
+    if !isdir(exercise_path)
```

Comment:
For readability, I'd prefer short-circuiting with `||` here, but that's personal preference. This works, too.

Reply:
I'll change it to a short circuit. Thanks!